de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/EditMetadataTemplateViewImpl.java

```diff
@@ -285,15 +286,15 @@ public String render(String object) {
             }
         });
         typeCombo.setTriggerAction(TriggerAction.ALL);
-        typeCombo.add(Arrays.asList("String",
-                                    "Timestamp",
-                                    "Boolean",
-                                    "Number",
-                                    "Integer",
-                                    "Multiline text",
-                                    "URL/URI",
-                                    "Enum"));
-        typeCombo.setValue("String");
+        typeCombo.add(Arrays.asList(MetadataTemplateAttributeType.STRING.toString(),
+                                    MetadataTemplateAttributeType.TIMESTAMP.toString(),
+                                    MetadataTemplateAttributeType.BOOLEAN.toString(),
+                                    MetadataTemplateAttributeType.NUMBER.toString(),
+                                    MetadataTemplateAttributeType.INTEGER.toString(),
+                                    MetadataTemplateAttributeType.MULTILINE.toString(),
+                                    MetadataTemplateAttributeType.URL.toString(),
+                                    MetadataTemplateAttributeType.ENUM.toString()));
```

Comment:
Not yet. It will be added here in a future PR along with additional changes required for Belphegor.

The Belphegor changes are going to be more complicated because we need to allow the admin to set some parameters for this OLS attribute type that can allow the DE ComboBox to set some LoadConfig parameters, such as restricting searches to only certain ontologies.